xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererVAAPIGL.cpp

```diff
@@ -75,9 +92,20 @@ bool CRendererVAAPI::Configure(const VideoPicture &picture, float fps, unsigned
   interop.glEGLImageTargetTexture2DOES = (PFNGLEGLIMAGETARGETTEXTURE2DOESPROC)eglGetProcAddress("glEGLImageTargetTexture2DOES");
   interop.eglDisplay = CRendererVAAPI::m_pWinSystem->GetEGLDisplay();
 
+  bool useVaapi2{}, hevc{};
+  VAAPI::CVaapi2Texture::TestInterop(pic->vadsp, CRendererVAAPI::m_pWinSystem->GetEGLDisplay(), useVaapi2, hevc);
```

Comment:
here we want to test either 10bit or 8bit. no need for testing both

Reply:
done